charts/apisix-ingress-controller/templates/configmap.yaml

```diff
@@ -58,7 +58,11 @@ data:
       {{ else }}
       default_cluster_base_url: http://{{ .Values.config.apisix.serviceName }}.{{ .Values.config.apisix.serviceNamespace }}.svc.{{ .Values.clusterDomain }}:{{ .Values.config.apisix.servicePort }}/apisix/admin
       {{- end}}
+      {{- if .Values.config.apisix.existingSecret }}
+      default_cluster_admin_key: "{{"{{"}}.APISIX_ADMIN_KEY{{"}}"}}"
```

Comment:
missing `$` prefix?

Reply:
not really apisix and ingress are using different approaches to access environment variables. Apisix uses a regular approach with $ENV_NAME styles, but ingress is based on go templates, which expects environment variables in {{.ENV_NAME}} format.